pipelines/creating-applications-with-cicd-pipelines.adoc

```diff
@@ -7,19 +7,19 @@ toc::[]
 
 With {pipelines-title}, you can create a customized CI/CD solution to build, test, and deploy your application.
 
-To create a full-fledged, self-serving CI/CD Pipeline for an application, you must perform the following tasks:
+To create a full-fledged, self-serving CI/CD pipeline for an application, you must perform the following tasks:
 
-* Create custom Tasks, or install existing reusable Tasks.
-* Create and define the delivery Pipeline for your application.
-* Create a PersistentVolumeClaim attached to the Workspace to provide the volume or filesystem for Pipeline execution.
-* Create a PipelineRun to instantiate and invoke the Pipeline.
-* Add Triggers to capture any events in the source repository.
+* Create custom tasks, or install existing reusable tasks.
+* Create and define the delivery pipeline for your application.
+* Provide a volume claim template, which creates a persistent volume claim attached to a workspace, or create a persistent volume claim. This persistent volume claim provides the storage volume or filesystem for the pipeline execution.
+* Create a `PipelineRun` object to instantiate and invoke the pipeline.
+* Add triggers to capture any events in the source repository.
 
 This section uses the `pipelines-tutorial` example to demonstrate the preceding tasks. The example uses a simple application which consists of:
 
 * A front-end interface, `vote-ui`, with the source code in the link:https://github.com/openshift-pipelines/vote-ui/tree/release-tech-preview-2[`ui-repo`] Git repository.
 * A back-end interface, `vote-api`, with the source code in the link:https://github.com/openshift-pipelines/vote-api/tree/release-tech-preview-2[`api-repo`]  Git repository.
-* The `apply_manifest` and `update-deployment` Tasks in the link:https://github.com/openshift/pipelines-tutorial/tree/release-tech-preview-2[`pipelines-tutorial`] Git repository.
+* The `apply_manifest` and `update-deployment` tasks in the link:https://github.com/openshift/pipelines-tutorial/tree/release-tech-preview-2[`pipelines-tutorial`] Git repository.
```

Comment:
Confirmation: Does one task use an `_` and the other a `-`?

Reply:
Excellent catch!!